distributed/scheduler.py

```diff
@@ -7818,6 +7818,20 @@ async def get_worker_logs(self, n=None, workers=None, nanny=False):
         return results
 
     def log_event(self, topic: str | Collection[str], msg: Any) -> None:
+        """Log an event under a given topic
```

Comment:
Do we need to test this as well?

Reply:
I'd appreciate an explicit test, that would make it easier to narrow down where things break.

Reply:
@hendrikmakait there's already a test that calls this: test_configurable_events_log_length and the only way a user should be calling the scheduler.log_event is via RPC which will have already checked if the message _is_dumpable so it doesn't need to be checked again

Reply:
Thanks for clarifying!